### ISSUE TITLE
Dagster DBT tutorial : 

### ISSUE THREAD

**Issue by clementlefevre** [dagster-io/dagster#14062]
### Dagster version

dagster, version 1.3.2

### What's the issue?

First, thanks all for this excellent project !
I wanted to try the **_dagster-dbt_** combo on my _Windows 11_ machine & _python 3.10.9_
In a fresh virtual env :
```
pip install dbt-core dagster dagit
```
then 
```
dagster project from-example --name tutorial_dbt_dagster --example tutorial_dbt_dagster
cd tutorial_dbt_dagster 
pip install -e ".[dev]"
```

then 
```
pip freeze
```
returns (extract):
```
dagit==1.3.2
dagster==1.3.2
dagster-dbt==0.19.2
dagster-duckdb==0.19.2
dagster-duckdb-pandas==0.19.2
dagster-graphql==1.3.2
dbt-core==1.5.0
dbt-duckdb==1.5.0
dbt-extractor==0.4.1
```

But when
```
cd tutorial_finished && dagster dev
```
i get the following output :
```
dagster.daemon.SchedulerDaemon - WARNING - Could not load location tutorial_dbt_dagster to check for schedules due to the following error: dagster._core.errors.DagsterImportError:
Encountered ImportError: `cannot import name 'DbtCliClientResource' from 'dagster_dbt'
```
Indeed, when looking into **\venv\Lib\site-packages\dagster_dbt\cli\resources.py** the class **DbtCliClientResource** is nowhere to be found (and is on **line 474** in the current **master** branch) . Could anyone be kind enough to point me where i got it wrong for the setup ? 
Cheers, CL.

### What did you expect to happen?

I wish the DbtCliClientResource existed in the installed dagster-dbt package.

### How to reproduce?

see the issue description.

### Deployment type

Local

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by yuhan** [comment]
@benpankow per https://github.com/dagster-io/dagster/pull/13816, this issue should be addressed with this week's 1.3.3 release right?

**Comment by clementlefevre** [comment]
OK, the Dagster 1.3.3 release solved the problem.

Again thanks to the team for this nice tool !
